### PR TITLE
Fix gallery thumbs navigation scrolling

### PR DESCRIPTION
**Pull request by iGerchak**
### Description (*)
Currently, we can't scroll thumbs navigation in the gallery on the product page on all touch devices.
This PR fixes this issue

### Fixed Issues (if relevant)

1. magento/magento2#21014: Gallery Thumbnail (left/right) Scroll Performance Android Chrome Sluggish and Unresponsive

### Preconditions (*)
Magento 2.3.2 | 2.3.3 
Product with a lot of images in the gallery

### Manual testing scenarios (*)
1. Go to edit the file: app/design/frontend/Magento/luma/etc/view.xml
2. Change **navtype** in the **Magento_Catalog -> gallery** from **slides** to **thumbs**
3. Save changes and flush Magento cache
4. Go to the frontend product page via mobile phone or tablet and try to scroll thumbs navigation

### Expected result (*)
1. Gallery thumbs navigation should be scrolled with Kinetic effect

### Actual result (*)
1. Gallery thumbs navigation scroll just a few pixels

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: lib/web/fotorama/fotorama.js [hunk 1]
```diff
@@ -2183,11 +2167,6 @@ fotoramaVersion = '4.6.4';
             if (o_allowFullScreen) {
                 $fullscreenIcon.prependTo($stage);
                 o_nativeFullScreen = FULLSCREEN && o_allowFullScreen === 'native';
-
-                // Due 300ms click delay on mobile devices
```
Comment (iGerchak) [inline lib/web/fotorama/fotorama.js:2187]: This bug was fixed in iOS 11

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-6368](https://jira.corp.magento.com/browse/ENGCOM-6368) has been created to process this Pull Request

File: lib/web/fotorama/fotorama.js (hunk 1 above)
Comment (krzksz) [reply lib/web/fotorama/fotorama.js:2187]: I'm not sure if we can remove it, as according to https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements_browsers.html Magento should support even iOS 7.

**Review by krzksz — CHANGES_REQUESTED** [review]

**Comment by ihor-sviziev** [comment]
@krzksz I believe this docs just outdated and that's not true.
I'll discuss it with other maintainters.

**Comment by ihor-sviziev** [comment]
@slavvka could you point me should we support iOS 11 or not?

**Comment by slavvka** [comment]
> @slavvka could you point me should we support iOS 11 or not?

@ihor-sviziev I am still waiting for a response from PO

**Comment by ihor-sviziev** [comment]
@slavvka any updates?

**Comment by slavvka** [comment]
@ihor-sviziev Yes, PO approved that we support iPhone 6 and iOS 12, iOS 7 isn't supported anymore

File: lib/web/fotorama/fotorama.js (hunk 1 above)
Comment (slavvka) [reply lib/web/fotorama/fotorama.js:2187]: The documentation is updated https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements_browsers.html

**Review by krzksz — APPROVED** [review]
Great work and a great news that we don't need to support iOS 11.

**Comment by magento-engcom-team** [comment]
Hi @krzksz, thank you for the review.
[ENGCOM-6368](https://jira.corp.magento.com/browse/ENGCOM-6368) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**